cmd/subnetcmd/deploy.go

```diff
@@ -416,9 +418,11 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	fee := network.GenesisParams().CreateBlockchainTxFee
+	var fee uint64
 	if createSubnet {
-		fee += network.GenesisParams().CreateSubnetTxFee
+		fee = network.GenesisParams().CreateSubnetTxFee
+	} else {
+		fee = network.GenesisParams().CreateBlockchainTxFee
```

Comment:
nit: can be simplified with 
``
fee := network.GenesisParams().CreateBlockchainTxFee
if createSubnet {
		fee = network.GenesisParams().CreateSubnetTxFee
		}